.env.example

```diff
@@ -0,0 +1,3 @@
+AWS_ACCESS_KEY_ID="YOUR_ACCESS_KEY_ID"
+AWS_SECRET_ACCESS_KEY="YOUR_SECRET_ACCESS_KEY"
+AWS_REGION="YOUR_REGION"
```

Comment:
what is this file for?

Reply:
A file example for the environment variables to use in the project, this to protect the sensitive data

Reply:
don't worry, there's no need to include this because it reveals too much about project confidential data

Reply:
I'll delete the file